src/lib/i18n/locales/languages.json

```diff
@@ -114,5 +114,9 @@
 	{
 		"code": "dg-DG",
 		"title": "Doge 🐶"
+	},
```

Comment:
Should be

```
},
	{
		"code": "hr-HR",
      	        "title": "Croatian"
	}
]
```

Reply:
Hey, what do you mean here? I did format it json like, see below.